components/barcodes/barcode/overview.md

```diff
@@ -144,6 +75,62 @@ The Barcode methods are accessible through it's reference.
 }
 ````
 
+## Parameters
+
+The Blazor Barcode provides various parameters that allow you to configure the component:
+
+| Parameter | Type | Description |
+| ----------- | ----------- | ----------- |
+| `RenderAs` | `enum` | Defines the preferred rendering mode - svg/canvas. |
+| `Checksum` | `bool` | By setting it to true, the Barcode will display the checksum digit next to the value in the text area. |
+| `Type` | `enum` | Defines the symbology (encoding) the Barcode will use - ([full list of supported encodings]({%slug  barcode-encoding%})) |
+| `Value` | `string` | Defines the initial value of the Barcode. |
+| `Width` | `string` | |
+| `Height` | `string` | |
+| `Class` | `string` | The CSS class that will be rendered on the main wrapping element of the Barcode component. |
+| `Background` | `string` | |
+| `Color` | `string` | Defines the color of the bar elements. |
+
+### BarcodeText parameters
+
+The following `BarcodeText` parameters enable you to customize the appearance of the Blazor Barcode text:
+
+| Parameter | Type | Description |
+| ----------- | ----------- | ----------- |
+| `Color` | `string` | |
+| `Font` | `string` | |
+| `BarcodeTextMargin` | `object` | |
+| `BarcodeTextMargin.Bottom` | `double` | |
+| `BarcodeTextMargin.Left` | `double` | |
+| `BarcodeTextMargin.Right` | `double` | |
+| `BarcodeTextMargin.Top` | `double` | |
+| `Visible` | `bool` | By setting it to false, the Barcode will not display the value as a text below the barcode lines. |
```

Comment:
The default value of `bool` is false, so we need to say it's `true` in this case.
```suggestion
| `Visible` | `bool` <br /> `true` | If `false`, the Barcode will not display the value as text below the barcode lines. |
```